db/go.mod

```diff
@@ -3,12 +3,10 @@ go 1.18
 module github.com/cosmos/cosmos-sdk/db
 
 require (
-	// Note: gorocksdb bindings for OptimisticTransactionDB are not merged upstream, so we use a fork
-	// See https://github.com/tecbot/gorocksdb/pull/216
-	github.com/cosmos/gorocksdb v1.2.0
 	github.com/dgraph-io/badger/v3 v3.2103.2
 	github.com/dgraph-io/ristretto v0.1.0
 	github.com/google/btree v1.0.1
+	github.com/linxGnu/grocksdb v1.7.1
```

Comment:
mentioned this already in other PRs, this is the second time we are changing this fork, lets converge on the cosmos one. Happy to upstream your work after review from the team working on storage reviews it.

Reply:
When we changed that, we unfortunately forked the **woefully unmaintained** rocksdb wrapper library for go.